frontend-react/src/pages/Upload.tsx

```diff
@@ -329,7 +329,7 @@ export const Upload = () => {
                                 return (
                                     <tr key={"error_" + i}>
                                         <td>{e["message"]}</td>
-                                        <td>{e["itemNums"]}</td>
+                                        <td>Rows: {e["itemNums"]}</td>
```

Comment:
I think there's opportunity to reduce confusion here a little
```suggestion
                                        <td>Row(s): {e["itemNums"]}</td>
```

Reply:
Sounds good to me!